CWE/decisions/Romania.txt

```diff
@@ -29,4 +30,195 @@ political_decisions = {
 			factor = 1
 		}
 	}
-}
\ No newline at end of file
+
+	oil_modernization_completion = {
+		picture = "rom_oil"
+		potential = {
+			AND = {
+				owns = 667
+				owns = 672
+				owns = 665
+			}
+			OR = {
+				has_country_flag = nicolae_ceausescu_flag
+				has_country_flag = rom_manescu
+				has_country_flag = gheorghe_apostol_romania
+				has_country_flag = rom_nicu
+				has_country_flag = rom_zoia
+			}
+			OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = socialist_democracy } }
+			has_country_flag = romanian_oil_modernization_start
+			NOT = { has_country_flag = romanian_oil_modernization_complete }
+		}
+		
+		allow = {
+			NOT = { has_country_modifier = economic_malaise }
+			war = no
+			money = 100000
+		}
+		
+		effect = {
+			prestige = 100
+			667 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 			
+			}
+			672 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 			
+			}
+			665 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 		
+			}
+			set_country_flag = romanian_oil_modernization_complete
+			treasury = -75000
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
+
+
+
+	oil_modernization_hungarian_region = {
```

Comment:
If this decision was meant to be taken under historical circumstances then this decision should be set as fired in Romania's country history file (under the 1992 section) using:

decision = oil_modernization_hungarian_region

Reply:
It is not, the modernization program never came to fruition except under privatized systems and on a much smaller scale well into the 90s iirc.

Reply:
Alright.